pkg/certificate/cleanup_test.go

```diff
@@ -322,37 +309,36 @@ var _ = Describe("CABundle cleanup", func() {
 					notBefore: -1 * time.Hour,
 					notAfter:  80 * time.Hour,
 				},
-				{
-					notBefore: -34 * time.Hour,
-					notAfter:  90 * time.Hour,
-				},
 			},
 		}),
-		Entry("All beyond overlap duration, should keep the latest appended", cleanUpCertificatesCase{
-			overlapDuration: 5 * time.Hour,
+		Entry("One certificate expired now, should remove it", cleanUpCertificatesCase{
```

Comment:
I think you have duplicate Entry here.

Reply:
I don't see it duplicated.

Reply:
I can't find it either now.. maybe my mistake sorry